README.md

```diff
@@ -0,0 +1,255 @@
+<div align="center">
```

Comment:
- [ ] Your project needs a `secret-key-base` created locally:

![image](https://github.com/Tracy-miranja/appoint-doctor-backend/assets/57421823/3e9ea242-cefc-4170-9593-ee3cd0bc858e)

Kindly, add the instructions to create it to your readme:

```
1. Remove config/master.key and config/credentials.yml.enc if they exist.
2. Run in the terminal: EDITOR=code rails credentials:edit 
3. Close the editor that opens.
4. This command will create a new master.key and credentials.yml.enc if they do not exist.
```

That way, the database creation will work for all your users. 👍